commonblocks/static/commonblocks/js/hallo-bootstrap.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/**
+ * registerHalloPlugin() applies to all editor instances of a Page.
+ * We want to encapsulate custom plugin configuration, per field.
+ */
+function makeHalloSimpleRichTextEditable(id, plugins) {
+    var defaultPlugins = $.extend({}, halloPlugins);
+
+    if (typeof plugins === 'object') {
+        $.each(plugins, function(plugin, options) {
+            registerHalloPlugin(plugin, options);
+        });
+
+        makeHalloRichTextEditable(id);
+    }
+
+    halloPlugins = $.extend({}, defaultPlugins);
```

Comment:
This, and the first line of the function, is only useful if the execution goes inside the `if` – so this should be inside the `if` too.